lib/ship_fosdick/downloader.rb

```diff
@@ -1,27 +1,21 @@
 module ShipFosdick
   class Downloader
-    def initialize(bucket)
-      @bucket = bucket
-    end
 
-    def download
+    def self.download
       [].tap do |returned_content|
         objects.each do |object|
           next unless object.key.downcase.include?('ship')
-          returned_content << object.content
+          next unless object.key.downcase.include?('.txt')
```

Comment:
This whole method seems like it could be condensed down to `objects.map.select`.

Also, what do you think about extracting the filter into a little policy object so it can be overridden or configured by the host app?